LottieDropper/Dropbox/DropboxBrowserViewModel.m

```diff
@@ -92,6 +109,21 @@ - (void)listFolderContinueWithClient:(DBUserClient *)client cursor:(NSString *)c
 	 }];
 }
 
+#pragma mark: - Fetch local files
+
+- (void) fetchLocalFiles {
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    NSURL *outputDirectory = [fileManager URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask][0];
+    NSError *error;
+    NSArray *directoryContent = [fileManager contentsOfDirectoryAtPath:outputDirectory.path error:&error];
+    // Create file for each content in the directoryContent and add it to the entries.
+    for (NSString *content in directoryContent) {
+        DBFILESMetadata * file = [[DBFILESMetadata alloc]initWithName:content];
```

Comment:
💀 Dit is zoals vermeld fout. Je moet een detailViewModel aanmaken hier onafhankelijk van DBFiles.